packages/psss-server/src/routes/reports/FetchArchivedAlertsReportRoute.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Route } from '../Route';
+
+export class FetchArchivedAlertsReportRoute extends Route {
```

Comment:
I notice that this route is very similar to the `FetchActiveAlertsReportRoute` route. Did you consider having one endpoint with a query param to set which type of data is being requested?

Reply:
I did, but again, these endpoints are just `input => fetch => output` - the only "complexity" is converting the `orgUnitCodes` param, so I thought that it would be more maintainable to define all logic in each file

Reply:
You know what, you are right - and actually that would reflect the original endpoint design:

Active alerts: /alerts
Archived alerts: /alerts/archived

Thanks for pointing this out - will change that!